api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java

```diff
@@ -1195,9 +1196,12 @@ public Concept getTrueConcept() {
 	public Concept getUnknownConcept() {
 		if (unknownConcept == null) {
 			try {
-				ConceptServiceImpl.setStaticUnknownConcept(Context.getConceptService().getConcept(
-				    Integer.parseInt(Context.getAdministrationService().getGlobalProperty(
-				        OpenmrsConstants.GLOBAL_PROPERTY_UNKNOWN_CONCEPT))));
+				Concept unknownConcept = Context.getConceptService().getConcept(
+					Integer.parseInt(Context.getAdministrationService().getGlobalProperty(
+						OpenmrsConstants.GLOBAL_PROPERTY_UNKNOWN_CONCEPT)));
+				initializeLazyPropertiesForConcept(unknownConcept);
+				
```

Comment:
Do you think there is any value in the original code, of setting the unknown concept via a new static method, instead of doing it just the same way as for the true and false concepts?

Reply:
I feel it would be better to do in the same way as the true and false concepts are done. However I don't know intention of creating a private stating method in the first time itself.
I can make this change if needed.